tests/hooks/useCompanyTags.test.jsx

```diff
@@ -107,4 +108,80 @@ describe('useCompanyTags', () => {
 
     expect(hookResponse.result.current.tagsByCompany).toEqual([])
   })
+
+  it('useCompanyTags when on save tags should call update tags service', async () => {
+    mockService(updateTags, { updated: true })
+    let hookResponse
+    await act(async () => {
+      hookResponse = renderHook(() => useCompanyTags(), { wrapper })
+    })
+
+    await act(async () => {
+      hookResponse.result.current.onSave()
+    })
+
+    expect(hookResponse.result.current.isLoading).toBeFalsy()
+    expect(updateTags).toBeCalled()
+  })
+
+  it('useCompanyTags when on save tags fails should set error message', async () => {
+    mockService(getCompanyTags, companyTagsResponse)
+    mockService(updateTags, 'error')
+    let hookResponse
+    await act(async () => {
+      hookResponse = renderHook(() => useCompanyTags(), { wrapper })
+    })
+
+    await act(async () => {
+      hookResponse.result.current.onSave()
+    })
+
+    expect(hookResponse.result.current.isLoading).toBeFalsy()
+    expect(hookResponse.result.current.tagsByCompany).toEqual(companyTagsResponse)
+    expect(hookResponse.result.current.error).toEqual(UPDATE_TAGS_ERROR)
+    expect(updateTags).toBeCalled()
+  })
+
+  it('useCompanyTags when on cancel tags update should set tags by company to the initial data ', async () => {
+    mockService(getCompanyTags, companyTagsResponse)
+    let hookResponse
+    await act(async () => {
+      hookResponse = renderHook(() => useCompanyTags(), { wrapper })
+    })
+
+    await act(async () => {
+      hookResponse.result.current.onCancel()
+    })
+
+    expect(hookResponse.result.current.tagsByCompany).toEqual(companyTagsResponse)
+    expect(updateTags).not.toBeCalled()
+  })
+
+  it('useCompanyTags when closing snackbar should not set error message', async () => {
```

Comment:
This test says that the error message isn't changed, but what we actually have is setting the error message to null, you can change this name to "should reset the default value of the error message" or "should remove the value of the error message".